go/client/logclient.go

```diff
@@ -308,3 +309,11 @@ func (c *LogClient) GetSTH() (sth *ct.SignedTreeHead, err error) {
 	sth.TreeHeadSignature = *ds
 	return
 }
+
+// GetSTHConsistency retrieves the consistency proof between two snapshots.
+func (c *LogClient) GetSTHConsistency(first, second uint64) ([][]byte, error) {
+	resp := new(getConsistencyProofResponse)
```

Comment:
I'd prefer `var resp getConsistencyProofResponse` (and declaring it at the "last moment"), especially since it's not returned (even if it was returned as a pointer, you could always have done `return &resp, err`, say).